src/modules/auth/org-not-linked-redirect.ts

```diff
@@ -17,6 +17,7 @@ import { Idp } from '~shared/auth/auth.types';
  * Which throws a 404 not found
  *
  * This way it works, but it also throws the error :s
+ * Note: Doesn't seem to work locally
```

Comment:
@bertyhell Can you have a look at this on your end?

Reply:
@ian-emsens-shd ik kon blijkbaar mijn workaround weg doen, want nu werkt een normale redirect wel:
https://github.com/viaacode/hetarchief-proxy/pull/257/commits/cdf6ca8e326f0997798869871e16708211a9a474